plugin/src/main/java/org/autorefactor/refactoring/rules/StringRefactoring.java

```diff
@@ -170,4 +174,43 @@ private boolean isStringValueOf(MethodInvocation node) {
                       || isMethod(node, "java.lang.String", "valueOf", "double")
                       || isMethod(node, "java.lang.String", "valueOf", "java.lang.Object"));
     }
+
+    @Override
+    public boolean visit(InfixExpression node) {
+        if (InfixExpression.Operator.PLUS.equals(node.getOperator())) {
+            final ASTBuilder b = this.ctx.getASTBuilder();

```

Comment:
(Do not act on this comment, I am fixing it)

Move this inside `maybeReplaceStringConcatenate()`.